contents/manual/events.mdx

```diff
@@ -116,6 +116,25 @@ If PostHog does not detect a type for an event property. You can choose any of t
 
 PostHog can capture all front-end events automatically using a small [JavaScript snippet](/docs/integrate). This means you do not need to add tracking to individual buttons, or parts of your product to get started at analyzing user behavior.
 
+### Capturing additional properties onto autocapture events
+
+If you add a data attribute onto an element in the format `data-ph-capture-attribute-some-key={someValue}`, then any autocapture event from that element or one of its children will have the property `some-key: 'someValue'` added to it. This can be useful when you want to add information about autocaptured events without needing to instrument a custom event. 
+
+For example, with a notifications bell:
```

Comment:
```suggestion
For example, with a notification bell:
```

I'm here to make the big edits.

Reply:
The skill is all in the little changes 🥇